umh.docs.umh.app/content/en/docs/getstarted/dataacquisitionmanipulation.md

```diff
@@ -25,10 +25,19 @@ The UMH includes 3 pre-configured data simulators for testing connections:
 OPC UA, often complex, can be streamlined using our Benthos-based OPC UA connector
 accessible from the Management Console.
 
+### Visualize Connections or Network Devices
+
+From the **Network View**, you can visualize the connections or network devices a UMH
```

Comment:
First, we need to explain what we mean with network devices and external services. A network device is a device that has an IP. Beause there can also be "virtual" devices that have an IP, should be monitred and should be dasta extracted from (e.g., an ERP system with a REST interface), we also mention external services directly. 

Maybe discuss with Paul the exact wording?